### PR TITLE
blackify

### PR DESCRIPTION
**Pull request by CBroz1**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,10 +3,14 @@
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
 
-## 0.0.0b2
+## [0.1.0] - 2022-05-10
```
Comment (kabilar) [inline CHANGELOG.md]:
```suggestion
## [0.1.0] - 2022-06-10
```

Comment (kabilar) [inline CHANGELOG.md]: Please add links to the bottom of the changelog.

```
[0.1.0]: https://github.com/datajoint/workflow-session/releases/tag/0.1.0
[0.0.0b2]: https://github.com/datajoint/workflow-session/tree/f91012c2391355d0b612757d55c15988636d0b9c
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,10 +3,14 @@
...
 ### Added
-+ Integration tests
++ Adopted black formatting into code base
+
 
+## [0.0.0b2] - 2022-03-07
```
Comment (kabilar) [inline CHANGELOG.md]:
```suggestion
## [0.0.0b2] - 2022-03-18 
```

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @CBroz1!  After this pull request is merged, please tag and manually release this version.

**Comment by CBroz1** [comment]
> Thanks @CBroz1! After this pull request is merged, please tag and manually release this version.

Will do

**Review by kabilar — APPROVED** [review]
Thanks @CBroz1!

**Comment by CBroz1** [comment]
Tagged 0.1.0
``` 
git tag 0.1.0
git push upstream 0.1.0
```

**Pull request merged**